viewing_party/party.py

```diff
@@ -1,23 +1,99 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if  title and genre and rating: 
+        return {
+            "title": title,
+            "genre": genre,
+            "rating": rating 
+        }
+    return None
+
+def add_to_watched(user_data, movie): 
+    if movie.items():
+        user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    if movie:
+        user_data["watchlist"].append(movie)
+    return user_data
 
+def watch_movie(user_data, title): 
+    if title not in [movie["title"] for movie in user_data["watchlist"]]:
+        return user_data
+    for value in user_data.values():
+        for movie in value: 
+            if movie["title"] == title: 
+                user_data["watched"].append(movie)
+                user_data["watchlist"].remove(movie)
+                return user_data

```

Comment:
You're over complicating this, you can simply loop through the watchlist and check if each movie title matches the title that is passed in. You can then remove that movie from the watchlist and add it to the watched list 😁